platform/build-faiss/build-prereqs.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+set -e
+export PATH=$HOME/.local/bin:$PATH
+export DEBIAN_FRONTEND=noninteractive
+
+sudo -E apt-get update && sudo -E apt-get dist-upgrade -y
+
+# Install python and build essentials and essential libraries
+sudo -E apt-get install -y python3-venv python3-pip python3-dev build-essential libssl-dev libffi-dev lib
+xml2-dev libxslt1-dev liblzma-dev libsqlite3-dev libreadline-dev libbz2-dev neovim curl git wget
+
+
+
+# Add a couple Python prerequisites
+pip install -U pip setuptools wheel
+pip install numpy swig torch
+
+export DEBIAN_FRONTEND=noninteractive
+
+# Get Intel OneAPI for BLAS support
+# From: https://www.intel.com/content/www/us/en/docs/oneapi/installation-guide-linux/2023-0/apt.html
+
+# download the key to system keyring
+wget -O- https://apt.repos.intel.com/intel-gpg-keys/GPG-PUB-KEY-INTEL-SW-PRODUCTS.PUB \
+| gpg --dearmor | sudo -E tee /usr/share/keyrings/oneapi-archive-keyring.gpg > /dev/null
+
+# add signed entry to apt sources and configure the APT client to use Intel repository:
+echo "deb [signed-by=/usr/share/keyrings/oneapi-archive-keyring.gpg] https://apt.repos.intel.com/oneapi all main" | sudo -E tee /etc/apt/sources.list.d/oneAPI.list
+
+
+sudo -E apt update
+sudo -E apt install dkms intel-basekit -y
```

Comment:
Why is `dkms` explicitly needed? I know `intel-basekit` installs drivers, although I don't think we require all of `OneAPI`. The part we do require does not possess drivers. Is the problem that `dkms` is an unspecified dependency by intel?